src/templates/legal/regulation.html.tt

```diff
@@ -0,0 +1,198 @@
+    <div>
+        <div class="background-gray center-text gr-padding-30 gr-12">
+            <h1 class="margin-top">[% l("Binary (IOM) Ltd") %]</h1>
+            <div class="gr-12">
+                <p>[% l("Binary (IOM) Ltd, First Floor, Millennium House, Victoria Road, Douglas, Isle of Man, IM2 4RW. Licensed and regulated by the Gambling Supervision Commission in the Isle of Man (current online gambling licence granted on the 31 August 2012) and for UK clients by the UK Gambling Commission - <a href=\"https://secure.gamblingcommission.gov.uk/gccustomweb/PublicRegister/PRSearch.aspx?ExternalAccountId=39172\" target=\"_blank\">view licence</a>.") %]</p>
+                <p>[% l("UK and Manx residents will have their account opened with Binary (IOM) Ltd.") %]</p>
+                <p><strong>[% l("Regulator:") %]</strong></p>
+                <div class="gr-2 gr-4-m gr-centered gr-padding-10">
+                    <a href="http://www.gov.im/gambling/" target="_blank">
+                        <img class="responsive" src="[% request.url_for('images/pages/footer/isle-of-man.png') %]">
+                    </a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="gray-line"></div>
+    <div class="center-text">
+        <img class="responsive timeline" src="[% request.url_for('images/pages/regulation/timeline.svg') %]">
+    </div>
+    <div>
+        <div class="background-gray center-text gr-padding-30 gr-12">
+            <h1 class="margin-top">[% l("Binary (C.R.) S.A.") %]</h1>
+            <div class="gr-12">
+                <p>[% l("Binary (C.R.) S.A., registered in Costa Rica. Clients from the rest of the world (excluding certain countries such as the USA, Costa Rica, Hong Kong) will have their account opened with Binary (C.R.) S.A.") %]</p>
+                <p>[% l("Binary.com is not available in Costa Rica and is not regulated or supervised by any regulatory body in the Costa Rican financial system.") %]</p>
+            </div>
+        </div>
+    </div>
+    <div class="gray-line"></div>
+    <div class="center-text">
+        <img class="responsive timeline" src="[% request.url_for('images/pages/regulation/timeline.svg') %]">
+    </div>
+    <div class="background-gray center-text gr-padding-30 gr-12">
+        <h1 class="margin-top">[% l("Binary (V) Ltd") %]</h1>
+        <div class="gr-12">
+            <p>[% l("Binary (V) Ltd, Govant Building, Port Vila, PO Box 1276, Vanuatu, Republic of Vanuatu. Licensed and regulated by the Vanuatu Financial Services Commission - <a href=\"https://www.vfsc.vu/wp-content/uploads/2015/12/List-of-Licensees-under-Dealers-in-Securities-Licensing-Act-CAP-70-09.09.20164.pdf\" target=\"_blank\">view licence</a>.") %]</p>
+            <p>[% l("Clients from the rest of the world (excluding certain countries such as the USA, Costa Rica, Hong Kong) will have their FX and CFD account opened with Binary (V) Ltd. Binary.com is not available in the Republic of Vanuatu.") %]</p>
```

Comment:
change "will have their FX and CFD account" to "will have their FX and CFD Metatrader 5 account"